CHANGES/35.feature

```diff
@@ -0,0 +1,2 @@
+Use flags for ``--simple`` and ``--structured`` on publication create instead of having users
+specify a value of ``True``.
```

Comment:
The file needs to be renamed to `CHANGES/36.feature` then the CI should finally be happy.

Reply:
Arg, I did rename it but forgot to add it to the commit when I amended it. Should be fixed now :)